README.md

```diff
@@ -5,6 +5,34 @@
 The project is still work in progress.
 Is possible to generate a working `danger-kotlin` instance from the code that is currently on this repo, but is not ready to be distributed yet.
 
+### What it looks like today
+You can make a Dangerfile that looks through PR metadata, it's fully typed.
+
+```kotlin
+import com.danger.dangerkotlin.*
```

Comment:
`systems.danger`?

Reply:
Yes I suppose is actually better given the website URL :P

Reply:
happy for `systems.danger`

Reply:
but i think we should do it in a separate PR

Reply:
let's merge everything what we have, then let's go with the migration to the new package name

Reply:
Yeah, sorry - was just thinking from a high level, no need to rush for it 👍